### PR TITLE
fixes double submit and adds code style fixes

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/report/report-download.component.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
         </select>
       </div>
       <div class="form-group mt-sm">
-          <button class="btn btn-info btn-sm btn-block" [disabled]="subscription != null" (click)="form.ngSubmit.emit()">{{'labels.reports.form.submit' | translate}}</button>
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/report/report-download.component.html]: Don't need the click handler now that the ```<button>``` is in the ```<form>```. If you have it it fires the submit twice

Comment (bgsavage) [inline webapp/src/main/webapp/src/app/report/report-download.component.html]: Can you explicitly set `type="submit"` so this doesn't get lost if another button was added here?

Comment (esotrope) [inline webapp/src/main/webapp/src/app/report/report-download.component.html]: noted - will make another pr for it
